app/facades/github_search_facade.rb

```diff
@@ -14,4 +14,12 @@ def get_followers
       Follower.new(follower)
     end
   end
+
+  def get_following
+    service = GithubService.new
+    following_data = service.get_following_data
+    following_data.map do |following|
+      Following.new(following)
+    end
+  end
```

Comment:
seems like the same lines are being repeated in same method. lets figure out a way to be more dry